aws.js

```diff
@@ -48,9 +48,9 @@ var AWS = (function() {
       var Crypto = loadCrypto();
       
       var d = new Date();
-      var dateStringFull =  String(d.getFullYear()) + addZero(d.getMonth()+1) + addZero(d.getDate()) + "T" + addZero(d.getUTCHours()) + addZero(d.getUTCMinutes()) + addZero(d.getUTCSeconds()) + 'Z';
```

Comment:
It must be return `20170613T233000Z` when timezone is `2017-06-14 08:30:00 +9:00`.
But, actually it returns `20170614T233000Z`. Because `Date#getDate()` is returns date with current timezone.